ClassySchedule/src/main/java/samples/Main.java

```diff
@@ -62,19 +63,113 @@ public void handle(ActionEvent e)
 
     }
 
-    public void login() {
+
+
+    public void adminLogin() {
 
 
         User test = new Admin("Bob", "Test", 123123,6,6,"easilyCrackable");
         Text text = new Text();
-        text.setText(test.getFirstName() + " " + test.getLastName());
+        text.setText(test.getFirstName() + " " + test.getLastName()+"'s homepage (Admin)");
 
         GridPane screen = new GridPane();
         screen.addRow(0, text);
-        Scene mainApplication = new Scene(screen, 500, 300);
+
+        // For adding new Users
+        screen.add(new Text(),0,1);
```

Comment:
I think we should create a new FXML file that has all the needed info on it instead of building the screen like this.